api/migrations/2024101114300_uk-stations.cjs

```diff
@@ -0,0 +1,32 @@
+/**
+ * Template for
+ * {@link https://salsita.github.io/node-pg-migrate/#/migrations?id=defining-migrations defining migrations}.
+ *
+ * @type {{
+ *   down: (pgm: import("node-pg-migrate").MigrationBuilder) => void | Promise<void>;
+ *   shorthands: Record<string, import("node-pg-migrate").ColumnDefinition>;
+ *   up: (pgm: import("node-pg-migrate").MigrationBuilder) => void | Promise<void>;
+ * }}
+ */
+const migration = {
+	up(pgm) {
+		pgm.createTable("uk_stations", {
+			station_id: { notNull: true, primaryKey: true, type: "int8" },
+			station_name: { notNull: true, type: "text" },
+			sixteen_character_name: { notNull: true, type: "text" },
+			longitude: { notNull: true, type: "float8" },
+			latitude: { notNull: true, type: "float8" },
+			crs_code: { notNull: true, type: "text" },
+		});
+		pgm.sql(
+			"INSERT INTO uk_stations (station_id, station_name, sixteen_character_name, longitude, latitude, crs_code) VALUES (1524, 'Manchester Piccadilly','MANCHESTER PIC',-2.228977818,53.47671998,'MAN')",
```

Comment:
Whilst this will work, it is a very strong convention to say Lat and Long.  Inserting data as Long and Lat is very likely to lead to data errors.

Reply:
I didn't understand it well. Do you mean instead of calling the column name latitude and longitude, it should be lat and long?

Reply:
No, I'm just saying that the order has a convention: you say latitude before longitude.  So (53.47, -2.22) rather than (-2.22, 53.47).  Get it the wrong way round and it is still a valid location, just the wrong one.

Reply:
ahh okay I'll fix it and push it again

Reply:
done